docs/user/openstack/connect_nodes_to_manila_network.md

```diff
@@ -0,0 +1,35 @@
+# Connecting nodes to a dedicated Manila network
+
+If you are going to use OpenStack Manila on your cluster, you should be aware that the service usually uses a standalone network for its shares, and by default the installer does not connect nodes to this network. It causes problems with mounting shares on an already deployed cluster, as there is no route to the network:
```

Comment:
>  the service usually uses a standalone network for its shares

Is it worth enumerating situations where this is not the case?

Reply:
Here is the deal.  CephFS via NFS back end (Red Hat supplied) will always need this extra network connection.  So will some vendor storage back ends.  But vendor backends support a mode that we in Maniila call DHSS=True where the NFS server presents itself directly on the tenant network (private network where the manila client resides).  In these cases no extra network is required.

@maxwelldb as you can see I don't have a crisp, succinct way to enumerate those situations but maybe you can craft something worth adding here ...